src/inmanta/server/extensions.py

```diff
@@ -134,20 +133,9 @@ def get_product_metadata(self) -> ProductMetadata:
             product="Inmanta Service Orchestrator",
             edition="Open Source Edition",
             license="Apache Software License 2",
-            version=self._get_product_version(),
+            version=__version__,
```

Comment:
I'm not entirely sure about this change. I like the simplicity, but it will always report the stable version. I'm not sure if that matters. @wouterdb wdyt?

Reply:
1. I like the simplicity
2. I do use this a lot to know the exact version installed (the exact dev version is often relevant on mgmt)
3.  I wonder if it would be worth it, that when building packages, use bumpversion to add the build tag to the version in `__version__`  or to add a `__tag__` as well to be able to know the full version, based purely on source?

Reply:
3. Could be interesting, but I honestly don't think bumpversion can handle it. Perhaps its replacement can, but I actually think it's considered out of scope. We could look into it.

Reply:
I would then revert this to use the previous implementation